content/docs/rfcs/61/README.md

```diff
@@ -0,0 +1,365 @@
+
+# Terminology
+
+The following terminology is used throughout this specification:
+
+| Name                 | References |
+| -------------------- | --- |
+| Waku node            | An Waku node ([10/WAKU2](/spec/10/)) that implements [11/WAKU2-RELAY](/spec/11/)|
+| Store node           | A Waku node that implements [13/WAKU2-STORE](/spec/13/) |
+| Waku network         | A group of Waku nodes forming a graph, connected via [11/WAKU2-RELAY](/spec/11/) |
+| Status user          | An Status account that is used in a Status consumer product, such as Status Mobile or Status Desktop |
+| Status node          | A Status client run by a Status application |
+| Control node      | A Status node that owns the private key for a Status community |
+| Community member     | A Status user that is part of a Status community, not owning the private key of the community |
+| Community member node| A Status node with message archive capabilities enabled, run by a community member |
+| Live messages        | Waku messages received through the Waku network |
+| BitTorrent client    | A program implementing the [BitTorrent](https://bittorrent.org) protocol |
+| Torrent/Torrent file | A file containing metadata about data to be downloaded by BitTorrent clients |
+| Magnet link          | A link encoding the metadata provided by a torrent file ([Magnet URI scheme](https://en.wikipedia.org/wiki/Magnet_URI_scheme)) |
+
+# Supported platforms
+
+Creating, distributing, downloading and unpacking community history archives SHOULD only be supported in Status desktop clients. Status mobile are out of scope for this version of the service. However we may consider bringing this service to Status Mobile in the future.
+
+This section MAY also specify endpoint behaviour when receiving specific messages, e.g. the behaviour of certain caches etc.
+
+# Requirements / Assumptions
+
+This specification has the following assumptions:
+
+- Store nodes are available 24/7, ensuring constant live message availability
+- The storage time range limit is 30 days
+- Store nodes have enough storage to persist historical messages for up to 30 days
+- No store nodes have storage to persist historical messages older than 30 days
+- All nodes are honest
+- The network is reliable
+
+Furthermore, it assumes that:
+
+- Control nodes have enough storage to persist historical messages older than 30 days
+- Control nodes provide archives with historical messages **at least** every 30 days
+- Control nodes receive all community messages
+- Control nodes are honest
+- Control knows at least one store node from which it can query historical messages if need be
```

Comment:
```suggestion
- Control nodes have enough storage to persist historical messages older than 30 days.
- Control nodes provide archives with historical messages **at least** every 30 days.
- Control nodes receive all community messages.
- Control nodes are honest.
- Control nodes know at least one store node from which it can query historical messages.
```